panels/user-accounts/cc-user-panel.ui

```diff
@@ -655,4 +667,17 @@
       <widget name="last_login_row"/>
     </widgets>
   </object>
+  <object class="GtkPopover" id="parental_controls_unavailable_popover">
+    <property name="can_focus">False</property>
+    <property name="border_width">12</property>
+    <child>
+      <object class="GtkLabel">
+        <property name="visible">True</property>
+        <property name="can-focus">False</property>
+        <property name="wrap">True</property>
+        <property name="label" translatable="yes">Parental controls can now only be applied to non-administrator accounts. See the &lt;a href="https://community.endlessos.com/t/how-have-parental-controls-changed-in-endless-os-3-8/12313"&gt;FAQ entry&lt;/a&gt;.</property>
```

Comment:
I worry slightly about translations of the URL but I think in practice we will have it in English and manually adjust the translations in other languages as and when someone translates it on the forum.